frontend/src/pages/Admin/Subcontractors/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Typography } from "@mui/material";
+
+export const Subcontractors = () => {
+  return (
+    <div style={{ height: "50vh", width: "200%" }}>
```

Comment:
Is there a reason we're going with inline styles for this? I realize this may have been copied over but usually we try to avoid inline styles unless they're strictly necessary